### PR TITLE
Add Vertiq Documentation to PX4

### PR DESCRIPTION
**Pull request by vertiq-jordan**
Adds Vertiq documentation to the PX4 Wiki. The content is the same as that found here: https://docs.google.com/document/d/1UFhBRvcGhGlJVy9aAXr6kjc0mA3kOuSZfsDIceyO4Is/edit

I've tested, and all links work as expected.

Some screenshots of the built version:
image
image
image
image

### CODE REVIEW

File: assets/peripherals/esc_vertiq/vertiq_esc_lineup.jpg
Comment (iq-fred) [inline assets/peripherals/esc_vertiq/vertiq_esc_lineup.jpg:1]: This is a test comment to try and become a reviewer

Comment (iq-fred) [inline assets/peripherals/esc_vertiq/vertiq_esc_lineup.jpg:1]: I noticed there is a 60 in this picture. Did you check with Jon if he is ok with having that show up here at this point?

File: en/peripherals/vertiq.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Where to Buy
+
+Purchasing information can be found on the [Vertiq website](https://www.vertiq.co/).
+
+
+## Hardware Setup
+### Wiring
+
+Connecting your Vertiq module to a PWM input or DroneCAN bus will vary depending on your model. Please see the product data sheets for wiring information. All Vertiq datasheets can be found at [vertiq.co](https://www.vertiq.co/).
```
Comment (iq-fred) [inline en/peripherals/vertiq.md]: Would it be a PWM input or a PWM output? Sort of a matter of perspective, but I guess if you are connecting the motor to it I would expect to be classified as an output?

File: assets/peripherals/esc_vertiq/vertiq_esc_lineup.jpg
Comment (vertiq-jordan) [reply assets/peripherals/esc_vertiq/vertiq_esc_lineup.jpg:1]: He's the one who took the picture for me for this doc, so I would assume yes

**Review by iq-fred — APPROVED** [review]

File: assets/peripherals/esc_vertiq/vertiq_esc_lineup.jpg
Comment (iq-fred) [reply assets/peripherals/esc_vertiq/vertiq_esc_lineup.jpg:1]: I did follow up with Jon on this, and he said he is fine with it

**Pull request merged**